modelling/src/test/java/org/axonframework/modelling/command/AbstractRepositoryTest.java

```diff
@@ -216,4 +216,13 @@ void checkedExceptionFromConstructorDoesNotAttemptToStoreAggregate() {
         assertTrue(uow.getExecutionResult().isExceptionResult());
         assertEquals("Throwing checked exception", uow.getExecutionResult().getExceptionResult().getMessage());
     }
+
+    private static DeadlineMessage<String> aDeadlineMessage() {
```

Comment:
Here, I use a convention—if it's okay with you—that it's a (not "the" or some specifically configured) deadline message. I don't care about the content of the message; it's just a fixture for testing purposes.
It could be aUser, aMessage, etc. - something random to be used in tests.

This was done intentionally. Please note that there is no argument to treat something specifically as a "deadline message."

Reply:
Aaah I completely missed that, sorry! Thanks for clarifying, @MateuszNaKodach!